doc/source/whatsnew/v0.23.1.txt

```diff
@@ -138,3 +138,4 @@ Bug Fixes
 
 - Tab completion on :class:`Index` in IPython no longer outputs deprecation warnings (:issue:`21125`)
 - Bug preventing pandas being used on Windows without C++ redistributable installed (:issue:`21106`)
+- Bug in :func:`Dataframe.asof` that raised a ``TypeError`` when attempting to compare tz-naive and tz-aware timestamps (:issue:`21194`)
```

Comment:
Could you move this entry to v0.24.0.txt?

Reply:
I have moved the entry to v0.24.0.txt, section "Timezones". I hope it is OK.

Reply:
Good choice!